packages/block-library/src/buttons/style.scss

```
@@ -5,7 +5,8 @@ $blocks-block__margin: 0.5em;
 	display: flex;
 	flex-direction: row;
 	flex-wrap: wrap;
-	column-gap: $blocks-block__margin;
+	column-gap: var(--wp--style--block-gap, $blocks-block__margin);
+	row-gap: var(--wp--style--block-gap, $blocks-block__margin);
```

This means the same thing:
```suggestion
	gap: var(--wp--style--block-gap, $blocks-block__margin);
```

---

Yep, sorry this is in a muddled state in progress as I'm exploring a couple different options. This in particular is a vestige from when I was working with independent row and column gaps and definitely will be changed.